docker-compose.client.yml

```diff
@@ -0,0 +1,72 @@
+version: '3.5'
```

Comment:
There are some requirements on the docker-compose version required to be installed. Can you clarify?

Reply:
I choose the same version and format of the docker-compose files of the root folder for other purposes.

Reply:
What i meant is that there should be a requirements section in the README where we explain what sw is needed to run this. For example, we need nodeJS v xxx, docker-compose v.xxx,... Again, this README needs to be enough for a user to spin a client without the need to go to the main README. All information required should be here.